tests/kernel/mem_protect/mem_protect/src/mem_domain.c

```diff
@@ -14,6 +15,36 @@ K_THREAD_STACK_DEFINE(mem_domain_1_stack, MEM_DOMAIN_STACK_SIZE);
 K_THREAD_STACK_DEFINE(mem_domain_2_stack, MEM_DOMAIN_STACK_SIZE);
 K_THREAD_STACK_DEFINE(mem_domain_6_stack, MEM_DOMAIN_STACK_SIZE);
 struct k_thread mem_domain_1_tid, mem_domain_2_tid, mem_domain_6_tid;
+struct k_mem_domain domain0, domain1, domain2, domain4, name_domain,
+		    overlap_domain;
+
+struct k_thread user_thread0, parent_thr, child_thr;
+k_tid_t usr_tid0, usr_tid1, parent_tid, child_tid;
+K_THREAD_STACK_DEFINE(user_thread0_stack, STACK_SIZE);
+K_THREAD_STACK_DEFINE(child_thr_stack, STACK_SIZE);
+K_THREAD_STACK_DEFINE(parent_thr_stack, STACK_SIZE);
+
+struct k_sem sync_sem;
+
+u8_t __aligned(MEM_REGION_ALLOC) buf0[MEM_REGION_ALLOC];
+K_MEM_PARTITION_DEFINE(part0, buf0, sizeof(buf0),
+		K_MEM_PARTITION_P_RW_U_RW);
+
+K_APPMEM_PARTITION_DEFINE(part1);
+K_APP_BMEM(part1) u8_t __aligned(MEM_REGION_ALLOC) buf1[MEM_REGION_ALLOC];
+struct k_mem_partition *app1_parts[] = {
+	&part1
+};
+K_APPMEM_PARTITION_DEFINE(part_arc);
```

Comment:
why is this called `part_arc`?

Reply:
Maybe it is some legacy when I had different partition names for each architecture.

Reply:
It should be `part_arch` (architecture)